### ISSUE TITLE
NVDA causes audio to fail in Win7 64 bit

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#686]
**Reported by Rughead on 2010-06-01 07:41**
On HP E9220Y with AMD Phenon2 quad core, NVDA randomly stops speaking from time to time. I have to usually reboot to get NVDA to work again in most cases. All other sounds on computer stop working at this point. Even JAWS will not launch when this happens. Sometimes I can unload NVDA, at which system sounds will come back. This problem occurs in all kinds of situations. Sometimes when only arrowing up and down in an HTML or txt document. It has even happened a few times while filling out this form. NVDA seems to work fine on my XP Pro system, but only has this problem on my Win7 64 bit system. Here is a copy of the log file when this last happened:
INFO - nvda (22:59:03):
Starting NVDA
INFO - core.main (22:59:04):
Config dir: C:\Users\CLH 2990\AppData\Roaming\nvda
INFO - config.save (22:59:08):
Configuration saved
INFO - core.main (22:59:08):
NVDA version 2010.1
INFO - core.main (22:59:08):
Using Windows version (6, 1, 7600, 2, '')
INFO - core.main (22:59:08):
Using Python version 2.6.4 (r264:75708, Oct 26 2009, 08:23:19) [v.1500 32 bit (Intel)](MSC)
INFO - core.main (22:59:08):
Using comtypes version 0.6.2
INFO - synthDriverHandler.setSynth (22:59:15):
Loaded synthDriver eloquence
INFO - core.main (22:59:15):
Using wx version 2.8.10.1 (msw-unicode)
INFO - braille.initialize (22:59:15):
Using liblouis version 1.8.0
INFO - braille.BrailleHandler.setDisplayByName (22:59:15):
Loaded braille display driver noBraille
INFO - core.main (22:59:16):
NVDA initialized
ERROR - unhandled exception (23:04:03):
Traceback (most recent call last):
  File "virtualBuffers__init__.pyo", line 224, in <lambda>
  File "virtualBuffers__init__.pyo", line 454, in onAction
  File "textInfos\offsets.pyo", line 257, in copy
  File "virtualBuffers__init__.pyo", line 83, in **init**
  File "textInfos\offsets.pyo", line 218, in **init**
  File "virtualBuffers__init__.pyo", line 104, in _getStoryLength
WindowsError: [1775](Error) A null context handle was passed from the client to the host during a remote procedure call
ERROR - scriptHandler.executeScript (23:10:45):
error executing script: <bound method Gecko_ia2.script_activatePosition of <virtualBuffers.gecko_ia2.Gecko_ia2 object at 0x0704AFD0>> with key (None, 'return')
Traceback (most recent call last):
  File "scriptHandler.pyo", line 100, in executeScript
  File "virtualBuffers__init__.pyo", line 670, in script_activatePosition
  File "virtualBuffers__init__.pyo", line 624, in _activatePosition
  File "virtualBuffers__init__.pyo", line 794, in shouldPassThrough
AttributeError: 'NoneType' object has no attribute 'states'

**Comment by leonardder** [comment]
Last report of this seems to date from 6 years ago. Unless someone knows about similar issues, I think this can be closed.

**Comment by fernando-jose-silva** [comment]
I have this problem and I am using nvda next 14019 and windows 7 64bits.
What I can help, I'm at ease.

**Comment by ehollig** [comment]
Could #1710 be related?

**Comment by fernando-jose-silva** [comment]
I believe so.

Enviado do meu iPhone

**Comment by ehollig** [comment]
@fernando-jose-silva, are you still experiencing this issue or can this issue be closed?

**Comment by fernando-jose-silva** [comment]
I still have this problem, I also reported in the 1710.
If the nvda is with any beeps the sound hangs, if you disable the beeps the sound works.
This occurs on windows 7 64bits, on machines with windows 10 no.

**Comment by Brian1Gaff** [comment]
I'm using Windows 7 pro with 64bit system and do not see this on inbuilt or 
usb sound.

I do sometimes notice some  clipping of the end of some synths speech, but 
its intermittent and hard to pin down. certainly not something to go mad 
over.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by Adriani90** [comment]
I think this is machine specific, we had some discussions on Gitter with @jcsteh and @dlee but we need a valuable log file to see where the problem actually comes from.

**Comment by fernando-jose-silva** [comment]
I'm sorry, but I will not be able to help in this case, this machine was from a job where I'm no longer working, I lost access to it, and on my personal machine with windows 10 I can not reproduce the problem.

 

De: Ethan Holliger <notifications@github.com> 
Enviada em: terça-feira, 14 de agosto de 2018 23:24
Para: nvaccess/nvda <nvda@noreply.github.com>
Cc: fernando-jose-silva <fernandoj.silva21@gmail.com>; Mention <mention@noreply.github.com>
Assunto: Re: [nvaccess/nvda] NVDA causes audio to fail in Win7 64 bit (#686)

 

@fernando-jose-silva <https://github.com/fernando-jose-silva> , according to #686 (comment) <https://github.com/nvaccess/nvda/issues/686#issuecomment-408595141>  we think this is machine specific. To verify this, we need a valuable log file to see where the problem actually comes from.

**Comment by josephsl** [comment]
Hi,

I see. Please reopen this issue if anyone experiences this in the future. Thanks.
